docs/fsharp/index.yml

```diff
@@ -86,18 +87,23 @@ landingContent:
             url: language-reference/symbol-and-operator-reference/index.md
           - text: "Functions"
             url: language-reference/functions/index.md
-          - text: "F# types"
+          - text: "Types"
             url: language-reference/fsharp-types.md
-  - title: "F# Tools"
+
+  - title: "F# tools"
     linkLists:
       - linkListType: reference
         links:
           - text: "F# Interactive"
             url: tools/fsharp-interactive/index.md
+          - text: "F# development tools"
+            url: https://dotnet.microsoft.com/languages/fsharp/tools
 
-  - title: "F# Core Library reference"
+  - title: "F# library reference"
     linkLists:
       - linkListType: reference
         links:
-          - text: "FSharp.Core API reference"
-            url: https://fsharp.github.io/fsharp-core-docs/
+          - text: "F# core library reference"
+            url: https://aka.ms/fsharp-core-reference
```

Comment:
What's the reason to make this an aka.ms link? We generally avoid them in docs since it's so easy to update the docs - https://review.docs.microsoft.com/en-us/help/contribute/links-how-to?branch=master#fwlinks-and-akams-links.

Reply:
I see. I'll revert

I would expect us to move https://fsharp.github.io/fsharp-core-docs to either https://fsharp.github.io/docs or https://docs.fsharp.org/FSharp.Core at some point, but we can update later